contracts/contracts/SimpleReadAccessController/simple_read_access_controller.cairo

```diff
@@ -0,0 +1,33 @@
+%lang starknet
+
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
+from starkware.cairo.common.bool import TRUE, FALSE
+
+from ocr2.interfaces.IAccessController import IAccessController
+from SimpleReadAccessController.library import simple_read_access_controller
+from SimpleWriteAccessController.library import simple_write_access_controller
```

Comment:
This import doesn't seem to be used in this specific file, is it required?

Reply:
Removed